internal/exec/shell_utils.go

```diff
@@ -261,13 +175,13 @@ func execTerraformShellCommand(
 			componentEnvList = append(componentEnvList, fmt.Sprintf("PS1=%s", result.String()))
 		}
 	}
+	log.Debug("Starting a new interactive shell where you can execute all native Terraform commands (type 'exit' to go back)",
+		"component", component,
+		"stack", stack,
+		"workingDirector", workingDir,
```

Comment:
cwd is a common abbreviation we can use 
```suggestion
		"cwd", workingDir,
```